<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
<<<<<<< HEAD
       <number>5</number>
=======
       <number>4</number>
>>>>>>> a180ecb30c8d5b423161984eb304c6fef3fc8500
      </property>
      <widget class="QWidget" name="menuScreen">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>30</number>
          </property>
          <property name="bottomMargin">
           <number>50</number>
          </property>
          <item>
           <widget class="QLabel" name="menuScreenTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>28</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
              <stylestrategy>PreferDefault</stylestrategy>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>DOMINO KINGDOMS</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="newGameButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>155151</width>
              <height>1515115</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>NEW GAME</string>
            </property>
            <property name="autoRepeatInterval">
             <number>97</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="optionsButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>OPTIONS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exitButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text">
             <string>EXIT</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="loginScreen">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="nameForm">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="roomCodeForm">
            <property name="text">
             <string>Room code:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTextEdit" name="textEditRoomCode"/>
          </item>
          <item row="0" column="1">
           <widget class="QTextEdit" name="textEditName"/>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="backLoginButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>BACK</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QTextEdit" name="hostGameNameForm"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Host game name:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="joinLoginButton">
            <property name="text">
             <string>JOIN</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="hostLoginButton">
            <property name="text">
             <string>HOST</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="optionsScreen">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="text">
           <string>VOLUME</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rulesOptionsButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string>GAME RULES</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backOptionsButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string>BACK</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rulesScreen">
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>211</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string>GAME RULES:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="backRulesButton">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>510</y>
          <width>231</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>BACK</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="mainScreen">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="NewGameScreen">
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>688</width>
          <height>521</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="verticalSpacing">
          <number>200</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="pbHost">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>17</pointsize>
            </font>
           </property>
           <property name="text">
            <string>HOST</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pbJoin">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>17</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>JOIN</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>Join game!</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <italic>true</italic>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>Start new game and invite your friends!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pbBackNewGame">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>500</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
